doc/source/installation.rst

```diff
@@ -97,6 +97,29 @@ Here's a summary of the variations:
 
 .. _ray-install-java:
 
+Installing a specific dependency variant
+----------------------------------------
+To get finer grained control over the size of your dependencies for your Ray application, Ray provides a few optional dependency installation specifications. 
+
+These variants can be installed with:
+
+.. code-block:: bash
+
+    pip install ray[variant_string_goes_here]
+
+Availiable variant strings include:
+
+* ``default``: same as ``pip install ray``, includes standard Ray deps, but not things like :ref:`Ray dashboard <ray-dashboard>`, :ref:`Ray Serve <rayserve>`, etc 
+* ``serve``: includes dependencies for :ref:`Ray Serve <rayserve>`
+* ``tune``: includes dependencies for :ref:`Ray Tune <tune-60-seconds>`
+* ``k8s``:  includes dependencies for running :ref:`Ray on Kubernetes <ray-k8s-deploy>` (k8s)
+* ``observability``: includes dependencies for monitoring with Grafana
```

Comment:
I believe there is also ray[rllib], which installs like a lot of things ...

Reply:
added!